.github/workflows/cpu-inference.yml

```diff
@@ -24,6 +24,8 @@ jobs:
   unit-tests:
     runs-on: [self-hosted, cpu]
 
+    env: {ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: true} # Allow using Node16 actions
```

Comment:
Would this also work with checkout@v4, i.e. force it to use node16? In that case, should we consider upgrading to v4 actions for all our workflows?

Reply:
I think checkout@v4 supports only node20, which is why we couldn't upgrade those workflows yet.  I think we should just work on getting those runners to use images that contain node20/node22/updated containers so we can remove this envvar entirely